Done all assignments.

GeneralVader opened the pull request:
did all questions

GeneralVader committed:
Done all assignments.

hex-plex commented on Week1/work.py:
```diff
@@ -66,8 +63,16 @@ def even_sort(arr):
+        else:
+            b=np.append(b,arr[i])
+    
+    return np.concatenate((e,b))
```
Though this is not fair on my side to tell, but do look into the expected type of output we have asked from the function and that you have used so -please use type casting and it should be fixed.

GeneralVader replied:
Okay bhaiyya I inserted the typecasting and committed the changes and now it checks all. Thank you and sorry for the inconvenience.

GeneralVader committed:
Did some typecasting for the output to match

GeneralVader committed:
Merge pull request #1 from AMC-IITBHU/main

Week 2

GeneralVader committed:
Merge pull request #2 from AMC-IITBHU/main

Pull req

GeneralVader committed:
Error after new test cases

changed the sqrt function to return complex on negative and changed palindrome to account for words starting with capital letter

GeneralVader committed:
Done 45 questions

GeneralVader commented:
done ques till 45+69th ques in week 2 assign
